input/fsh/EX_Questionnaire.fsh

```diff
@@ -62,7 +62,11 @@ Usage: #example
 * item[=].text = "Auftrag"
 * item[=].type = #group
 * item[=].required = true
-* item[=].item[0].linkId = "order.placerOrderIdentifier"
+* item[=].item[0].linkId = "order.authoredOn"
+* item[=].item[=].definition = "http://build.fhir.org/ig/hl7ch/ch-orf/StructureDefinition-ch-orf-servicerequest.html#ServiceRequest.authoredOn"
```

Comment:
@JBleuer : this should not be http://build.fhir.org/ig/hl7ch/ch-orf/ but start with http://fhir.ch/ig/ch-orf, can you fix all occurrences?

Reply:
Fixed